application/event_processor/tests/test_reporting.py

```diff
@@ -200,6 +204,7 @@ def test_log_service_with_generic_bank_holiday(mock_logger):
             "dos_service_type_id": dos_service.typeid,
             "bank_holiday_opening_times": OpenPeriod.list_string(open_periods),
             "nhsuk_parentorg": nhs_entity.parent_org_name,
+            "dos_service_type_name": nhs_entity.service_type,
```

Comment:
This is wrong! The service type variable is an Alias and not the official DoS service type. It is only meant to use used to determine what service type for development/debugging